samples/subsys/logging/kernel_event_logger/sample.yaml

```diff
@@ -4,6 +4,10 @@ sample:
   name: Kernel event logger Sample
 tests:
   test:
-    platform_whitelist: qemu_x86
```

Comment:
maybe a note in the commit message on why we can remove the whitelist

Reply:
done